CONTRIBUTE

```diff
@@ -0,0 +1,69 @@
+CONTRIBUTING TO TQDM
```

Comment:
A procedure is missing to tell exactly what is needed in order to make the release.